src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -139,4 +139,33 @@ public int getTotalMessageCount() {
     PreparedQuery results = datastore.prepare(new Query("Message"));
     return results.countEntities(FetchOptions.Builder.withLimit(1000));
   }
+
+  //Stores the user data
+  public void storeUser(User user) {
+	  Entity userEntity = new Entity("User",user.getEmail());
+
+	  userEntity.setProperty("email",user.getEmail());
+	  userEntity.setProperty("aboutMe",user.getAboutMe());
+	  datastore.put(userEntity);
+
+
+  }
+
+  //Returns the User owned by the email address
+
```

Comment:
remove line break here and change to JavaDoc style mentioned above, I think running the linter mentioned in our group chat will help with the minor style errors.

https://github.com/google/google-java-format

Reply:
Fixed